### ISSUE TITLE
Accessibility question

### ISSUE THREAD

**Issue by itsrogil** [json-editor/json-editor#997]
### General informations

- json-editor version: 2.5.3

### Question

Hello, we use `json-editor` at work and it's working as intended for our use so first of all, thank you for this. The only issue we have with it so far is that it raises accessibility flags because the input lack labels or if they are present, they don't have an id or an aria-label. [More infos here](https://dequeuniversity.com/rules/axe/4.1/label?application=axeAPI).

Is this something you might be currently working on and if not, would that be okay for me to do a pull request about it? It seems this project is fairly maintened but I just wanted to make sure before I spend time on this.

Thank you!

**Comment by schmunk42** [comment]
We're not working on this, PRs welcome!

**Comment by itsrogil** [comment]
> We're not working on this, PRs welcome!

I made a [PR for this](https://github.com/json-editor/json-editor/pull/1005) a couple of days ago, feel free to check it out @schmunk42 :)

**Comment by cbonaudo** [comment]
Hello, any news on this issue ? I also might need this feature and it looks like itsrogil's pr is neat and could solve my problem.

**Comment by schmunk42** [comment]
I have been on vacation, will check it...

**Comment by schmunk42** [comment]
fixed in #1005
